board/safety.h

```diff
@@ -486,6 +486,16 @@ float interpolate(struct lookup_t xy, float x) {
   return ret;
 }
 
+// Safety checks for longitudinal actuation
+bool long_accel_checks(int desired_accel, const LongitudinalLimits limits, const bool longitudinal_allowed) {
```

Comment:
prefer a little verbosity here for readability
```suggestion
bool longitudinal_accel_checks(int desired_accel, const LongitudinalLimits limits, const bool longitudinal_allowed) {
```